server/services/deviceService.ts

```diff
@@ -26,15 +10,52 @@ export default class DeviceService {
 
   async findByDeviceWearer(accessToken: string, deviceWearerId: string): Promise<Array<Device>> {
     try {
+      const dummyResponse = false
+      if (dummyResponse) {
+        return [
+          {
+            deviceId: '8225d883-0fd1-4456-aeba-f7701412d35e',
+            deviceType: 'RF',
+            status: 'Not fitted',
+            dateTagFitted: 'yesterday',
+            dateTagRemoved: 'today',
+          },
+          {
+            deviceId: '8225d883-0fd1-4456-aeba-f7701412d35a',
+            deviceType: 'RF',
+            status: 'Not fitted',
+            dateTagFitted: '2008-11-10',
+            dateTagRemoved: '2022-05-05',
+          },
+        ]
+      }
```

Comment:
Is this just for development/debugging? Should it be merged to main?

Reply:
Removing